src/common/config_parser.cpp

```diff
@@ -79,7 +79,8 @@ void ConfigParser::addOptionsGeneral(cli::CLIWrapper& cli) {
   cli.add<bool>("--relative-paths",
      "All paths are relative to the config file location");
   cli.add<std::string>("--dump-config",
-     "Dump current (modified) configuration to stdout and exit. Possible values: full, minimal")
+     "Dump current (modified) configuration to stdout and exit. "
+     "Possible values: full, minimal, explain")
```

Comment:
Add comment: what do the values mean. I think we already saw that "minimal" is actually a bit counterintuitive.
Please explain "explain". Is that a good name?